### PR TITLE
Adds a xenobio wiki book

### PR DESCRIPTION
**Pull request by MajManatee**
# Document the changes in your pull request

Adds a book to xenobiology, allowing you to not have to spend a year and a half trying to type out "feed slime" but in much more complex terms.

# Wiki Documentation

The book needs to be added somewhere.

# Changelog

:cl:  
rscadd: Adds a wiki book to xenobio, to help teach new people.
/:cl:

### CODE REVIEW

**Comment by alexkar598** [comment]
Watching in order to move protect the page on merge

**Comment by MajManatee** [comment]
idk what that means, but... thanks?

**Comment by JamieD1** [comment]
@grimm-hollowknight

**Review by nesisknesis — APPROVED** [review]

**Comment by alexkar598** [comment]
> idk what that means, but... thanks?

It means flagging the page in sort that noone can delete or rename it without my approval to prevent the book from breaking

File: yogstation/code/game/objects/items/manuals.dm [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/obj/item/book/manual/wiki/xenobiology
+	name = "Crossbreeding: A Simpletons guide to slimey stuff."
```
Comment (Anvilman6) [inline yogstation/code/game/objects/items/manuals.dm]: name = "Crossbreeding: The Definitive Guide to Slimy Stuff"

File: yogstation/code/game/objects/items/manuals.dm [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+	icon = 'yogstation/icons/obj/library.dmi'
+	icon_state = "xenobook"
+	author = "Dr. S. Lime"
+	title = "Crossbreeding: A simpletons guide to slimey stuff."
```
Comment (Anvilman6) [inline yogstation/code/game/objects/items/manuals.dm]: title = "Crossbreeding: The Definitive Guide to Slimy Stuff"

**Review by Anvilman6 — CHANGES_REQUESTED** [review]
Title change suggestion, make sure that the name and title fields are unified in either case

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
merge conflict

**Comment by MajManatee** [comment]
i HATE merge conflicts

**Pull request closed without merging**